cf/test/create_test_files_2.py

```diff
@@ -0,0 +1,10440 @@
+        "time_1_bounds", "f4", ("time_1", "bounds2")
+    )
+    time_1_bounds[...] = [0, 30]
+
+    height = n.createVariable("height", "f8", ())
+    height.units = "m"
+    height.standard_name = "height"
+    height.positive = "up"
+    height.axis = "Z"
+    height[...] = 2
+
+    src = n.createVariable("src", "f8", ("time", "latitude", "longitude"))
+    src.standard_name = "air_temperature"
+    src.units = "K"
+    src.coordinates = "height"
+    src.cell_methods = "time: mean"
+
+    # Don't generate this data randomly - it's useful to see the real
+    # patterns of global temperature.
+    src[...] = [
+        [
+            [
+                243.6,
+                242.4,
+                241.8,
+                241.5,
+                241.2,
+                240.8,
+                240.5,
+                240.4,
+                240.2,
+                240.5,
+                241.2,
+                241.9,
+                242.5,
+                243.0,
+                243.4,
+                244.1,
+                245.2,
+                246.4,
+                247.0,
+                247.4,
+                248.3,
+                250.2,
+                251.6,
+                252.3,
+                253.9,
+                255.8,
+                257.6,
+                258.7,
+                258.5,
+                257.7,
+                256.8,
+                255.6,
+                254.1,
+                252.0,
+                251.6,
+                252.4,
+                254.0,
+                255.1,
+                254.8,
+                254.3,
+                253.4,
+                252.6,
+                251.5,
+                249.9,
+                248.3,
+                246.4,
+                244.9,
+                244.2,
+            ],
+            [
+                243.3,
+                241.0,
+                239.8,
+                238.3,
+                237.4,
+                237.4,
+                238.9,
+                240.3,
+                241.3,
+                241.2,
+                241.2,
+                240.8,
+                241.0,
+                242.6,
+                244.4,
+                245.5,
+                246.4,
+                248.2,
+                249.8,
+                251.0,
+                254.3,
+                260.7,
+                265.4,
+                266.1,
+                265.9,
+                265.9,
+                266.4,
+                266.1,
+                265.3,
+                264.2,
+                262.8,
+                261.8,
+                261.0,
+                258.7,
+                256.9,
+                257.3,
+                259.5,
+                262.7,
+                264.4,
+                264.9,
+                265.4,
+                264.2,
+                264.7,
+                263.4,
+                258.3,
+                251.8,
+                247.2,
+                245.4,
+            ],
```

Comment:
Why is this new file so large by line count, I wondered, and then saw that (`black`/PEP8) makes it sparse like this and it all made sense :grin: I guess these `create_test_file*.py` pair files being 10K+ lines is just about manageable, but we could consider moving the hard-coded large written-out data arrays into a separate file or files to read-in, perhaps using the [`numpy.savetxt`](https://numpy.org/doc/stable/reference/generated/numpy.savetxt.html) and `numpy.loadtxt` functions, else maybe `csv` format or `pickle`? It would probably be quite useful to be able to see the logical code in such modules without having to wade through raw numbers...

Reply:
Good idea - I'll look into it.

Reply:
See b1b374ad5972335bb11be6fdf301d7ecf9ee805f

Reply:
Excellent stuff. Nice idea/knowledge with these `npz` files, I hadn't heard of them before but they look perfect! Thanks, marking as resolved.